server.js

```diff
@@ -17,6 +12,65 @@ mongoose.Promise = Promise;
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
+
+const Song = mongoose.model("Song", {
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+
+})
+// The reset of database and adding new songs or albums 
+// was great for educational purpose but is not to be used live.
+//
```

Comment:
Good explanation of why you have the RESET_DB, it shows that you understand the purpose of it.